e2e/index.js

```diff
@@ -0,0 +1,40 @@
+const path = require('path')
+const fixtures = require('./fixtures')
+const {Ganache} = require('./ganache')
+const ReferenceCounter = require('./rc')
+const {buildWebDriver, Driver} = require('./webdriver')
+
+const fixtureServer = new fixtures.FixtureServer()
+const rc = new ReferenceCounter(async () => fixtureServer.start(), async () => fixtureServer.stop())
+
+const load = async function load (fixturesDirectory, callback) {
+  await rc.tick(async () => {
+    const browser = String(process.env.BROWSER || 'firefox')
+    const extensionPath = path.resolve(__dirname, `../dist/${browser}`)
+    const {driver, ganache} = await fixtureServer.loadState(fixturesDirectory, async ({ganacheOptions}) => {
+      const ganache = new Ganache()
+      await ganache.start(ganacheOptions)
+      const {driver, extensionUrl} = await buildWebDriver({
+        browser,
+        extensionPath,
+      })
+
+      await driver.get(extensionUrl)
+
+      return {
+        driver,
+        ganache,
+      }
+    })
+    await callback({
+      ganache,
+      driver: new Driver(driver),
+    })
+    await driver.quit()
+    await ganache.quit()
+  })
+}
+
+module.exports = {
+  load,
```

Comment:
Can we give `load` a slightly more unique name so that it is easy to grep for its uses in the codebase?

Reply:
That's... a really good suggestion. I'll do `withFixtures`?